src/components/TabDevelop.vue

```diff
@@ -41,11 +41,11 @@
 
 <script lang="ts">
 import { Component, Prop, Vue, Watch } from "vue-property-decorator";
-// import { MdButton, MdContent, MdTabs } from "vue-material/dist/components";
+import { MdButton, MdContent, MdTabs } from "vue-material/dist/components";
```

Comment:
is this supposed to be here since we removed vue-material?

Reply:
I also deleted TabDevelop since we never use this file.